src/components/AllArticlesSortBy.jsx

```diff
@@ -0,0 +1,72 @@
+import { useState } from "react";
+import { Button } from "react-bootstrap";
+
+const AllArticlesSortBy = ({ setSearchParams }) => {
+  const [menuParams, setMenuParams] = useState({
+    sort_by: "",
+    order: "asc",
+  });
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setMenuParams({ ...menuParams, [name]: value });
+  };
+
+  const handleClick = () => {
+    const paramsString = `sort_by=${menuParams.sort_by}&order=${menuParams.order}`;
+    const searchParams = new URLSearchParams(paramsString);
+
+    if (menuParams.sort_by === "") {
+      alert("Both search parameters required");
+    } else {
+      setSearchParams(searchParams);
+    }
+  };
+
+  const style = {
+    fontSize: "20px",
```

Comment:
Unless there's a good reason for it not to be, this should be in your CSS file please!